functions/test/produce.test.js

```diff
@@ -12,12 +12,12 @@ describe("POST /produce", () => {
     request(app)
       .post("/produce")
       .send(produce)
-      .expect((res) => {
-        expect(res.text).not.toBeNull();
+      .expect((response) => {
+        expect(response.text).not.toBeNull();
       })
-      .end(function (err, res) {
-        if (err) return done(err);
-        id = JSON.parse(res.text).id;
+      .end((error, response) => {
+        if (error) return done(error);
```

Comment:
nit: would be good to put { } around these if statements, but _technically_ they are not needed